helm/csi-powerstore/values.yaml

```diff
@@ -114,7 +114,7 @@ controller:
     # Allowed values: Number followed by unit (s,m,h)
     # Examples: 60s, 5m, 1h
     # Default value: 60s
-    volumeHealthMonitorInterval: 60s
+    healthMonitorInterval: 60s
```

Comment:
why can't we rename it to just "interval", instead of "healthMonitorInterval", as this param is under already the "healthMonitor" section?
Also, this convention is followed in other drivers like [csi-powerscale](https://github.com/dell/csi-powerscale/blob/main/helm/csi-isilon/values.yaml), and we want to have config files to be consistent across all drivers.